rclcpp/minimal_action_server/README.md

```diff
@@ -1,3 +1,7 @@
 # Minimal action server cookbook recipes
 
-This package contains a few examples which show how to create action servers.
+This package contains an example which shows how to create an action server.
+
+"Composable" refers to the ability for a node to be used as part of a Composition (running multiple nodes in a single process).
+
+Note that not_composable.cpp instantiates a rclcpp::Node without subclassing it. This was the typical usage model in ROS 1, but unfortunately this style of coding is not compatible with composing multiple nodes into a single process. Thus, it is no longer the recommended style for ROS 2.
```

Comment:
> I just copied this paragraph from the subscriber README also located in this repo:

Actually that is only partially correct. You copied the rendered output but not the actual source. The source you tried to copy does follow the developer guide as well as uses markup to format specific parts of the sentence: https://raw.githubusercontent.com/ros2/examples/af08e6f7ac50f7808dbe6165f1adfd8e6cd3a79c/rclcpp/minimal_subscriber/README.md

The phrase `unfortunately` is certainly a personal opinion but since you closed the PR I won't bother about it either.